package.json

```diff
@@ -75,6 +74,7 @@
     "pngjs": "^3.0.0",
     "postcss-cli": "^5.0.0",
     "postcss-inline-svg": "^3.1.1",
+    "potpack": "^1.0.1",
```

Comment:
Shouldn't this be a regular dependency rather than a dev dependency?

Reply:
Yeah, my misstep, although it actually doesn't affect anything because GL JS is only usable with the built bundle. We might want to move most other regular deps to devdeps then.